### ISSUE TITLE
[xmpp] delivery acknowledgement broken for group chats

### ISSUE THREAD

**Issue by pravi** [loqui/im#734]
It always show red border, grey background. On both my server and conference server individual chats show green tick marks.

**Comment by TitanNano** [comment]
yes I'm aware of this. It's some issue in the muc plug-in for Strope.
